rllib/examples/centralized_critic.py

```diff
@@ -198,6 +207,8 @@ def get_policy_class(config):
     config = {
         "env": TwoStepGame,
         "batch_mode": "complete_episodes",
+        # Use GPUs iff `RLLIB_FORCE_NUM_GPUS` env var set to > 0.
+        "num_gpus": RLLIB_FORCE_NUM_GPUS,
```

Comment:
Instead of "num_gpus": RLLIB_FORCE_NUM_GPUS, consider explicitly inlining the logic here for reader clarity:
```
"num_gpus": int(os.environ.get("RLLIB_NUM_GPUS"), "0"),
```